code/core/arm9/source/main.cpp

```diff
@@ -307,6 +307,12 @@ static void setupJit()
     jit_init();
 
     const auto& runSettings = gAppSettingsService.GetAppSettings().runSettings;
+    if (!runSettings.enableJit)
```

Comment:
Let's invert this if and move `jit_disable();` to the else. It should make the code easier to read